### ISSUE TITLE
Bug: random POST failures

### ISSUE THREAD

**Issue by palkra** [upb-uc4/hlf-network#32]
The CI sometimes randomly crashes with the following error message:

```
Error: POST failure of request: POST https://172.17.0.3:30905/enroll
{"hosts":["fv-az117"],"certificate_request":"-----BEGIN CERTIFICATE REQUEST-----\nMIIBRTCB7AIBADBkMQswCQYDVQQGEwJVUzEXMBUGA1UECBMOTm9ydGggQ2Fyb2xp\nbmExFDASBgNVBAoTC0h5cGVybGVkZ2VyMQ8wDQYDVQQLEwZGYWJyaWMxFTATBgNV\nBAMTDHRscy1jYS1hZG1pbjBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABKP7UJil\ndBAbiusjPsN9tUPJ/6G4NCRzBzV0C0Ux/HmqWieelpx8opgshx0RhEbe66XMXrM/\np308MWe6U/yFO6+gJjAkBgkqhkiG9w0BCQ4xFzAVMBMGA1UdEQQMMAqCCGZ2LWF6\nMTE3MAoGCCqGSM49BAMCA0gAMEUCIQCBOH6IfDjNiUnsScRmpzWv10IxxKHQ0zGO\nw4Yr1CcwyQIge4oIZtAQ6hlEZrEXgiUHE+Lnz/D0oheq9OxbwH/DcSY=\n-----END CERTIFICATE REQUEST-----\n","profile":"","crl_override":"","label":"","NotBefore":"0001-01-01T00:00:00Z","NotAfter":"0001-01-01T00:00:00Z","CAName":""}: Post https://172.17.0.3:30905/enroll: dial tcp 172.17.0.3:30905: connect: connection refused
##[error]Process completed with exit code 1.
```

**Comment by palkra** [comment]
Possible causes: 
 - Container is started but not yet listening

**Comment by NikoBergemann** [comment]
[where the action fails](https://github.com/upb-uc4/hyperledger_api/runs/1021043916?check_suite_focus=true)

**Comment by palkra** [comment]
Closed since we moved away from minikube in #68 and the bug does not occur using KinD.
